frontend/src/scenes/authentication/signup/signupForm/signupLogic.ts

```diff
@@ -95,10 +96,14 @@ export const signupLogic = kea<signupLogicType>([
                         first_name: payload.name.split(' ')[0],
                         last_name: payload.name.split(' ')[1] || undefined,
                         organization_name: payload.organization_name || undefined,
+                        role_at_organization: payload.role_at_organization || undefined,
```

Comment:
Shouldn't need this since it's required. The other fields are there to manipulate the value to undefined if it's not set.